vit_pytorch/__init__.py

```diff
@@ -1 +1,3 @@
 from vit_pytorch.vit_pytorch import ViT
+from vit_pytorch.mpp_loss import MPPLoss
```

Comment:
You can just place the `MPPLoss` into the same file as `mpp_pytorch`, since it likely will not be reused

Reply:
Sounds good